cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -522,6 +522,123 @@ <h2>Cash advances</h2>
     {% else %}
         <p>This card does not offer cash advances.</p>
     {% endif %}
+    <h2>Fees</h2>
+    <dl class="m-list m-list__card-details">
+        {% if not card.periodic_fee_type %}
+            <dt>Annual fee</dt>
+            <dd>None</dd>
+        {% endif %}
+        {% if card.annual_fee %}
+            <dt>Annual fee</dt>
+            <dd>{{ '$' ~ card.annual_fee }}</dd>
+        {% endif %}
+        {% if card.monthly_fee %}
+            <dt>Monthly fee</dt>
+            <dd>{{ '$' ~ card.monthly_fee }}</dd>
+        {% endif %}
+        {% if card.weekly_fee %}
+            <dt>Weekly fee</dt>
+            <dd>{{ '$' ~ card.weekly_fee }}</dd>
+        {% endif %}
+        {% if card.other_periodic_fee_amount %}
+            <dt>{{ card.other_periodic_fee_name }}</dt>
+            <dd>{{ '$' ~ card.other_periodic_fee_amount ~ ', ' ~ card.other_periodic_fee_frequency | lower }}</dd>
+        {% endif %}
+        {% if card.fee_varies %}
+            <dt>Fee varies?</dt>
+            <dd>
+                {{ '$' ~ card.periodic_min ~ '–'  ~ card.periodic_max ~
+                    ' (' ~ card.fee_explanation | lower ~ ')'
+                }}
+            </dd>
+        {% endif %}
+        <dt>Late fee</dt>
+        <dd>
+            {% if card.late_fees %}
+                {{ '$' ~ card.late_fee_dollars ~ ' for the first late payment' if
+                    card.late_fee_dollars
+                }}
+                {{ 'and' if card.late_fee_dollars
+                    and card.late_fee_six_month_billing_cycle
+                }}
+                {{ '$' ~ card.late_fee_six_month_billing_cycle ~
+                    ' for a late payment within six billing cycles of another
+                    late payment' if card.late_fee_six_month_billing_cycle
+                }}
+                {# A handful of cards specify neither a first nor repeat late
```

Comment:
Thank you for the descriptive comments! Very helpful to understand what is being shown.